resources/public/api/conf/2.0/application.raml

```diff
@@ -72,6 +72,19 @@ documentation:
                   example: 4557ecb5-fd32-48cc-81f5-e6acd1099f3c
               get: !include uk_property_period_summary_retrieve.raml
               put: !include uk_property_period_summary_amend.raml
+  /non-furnished-holiday-lettings:
+    /{nino}:
+      uriParameters:
+        nino:
+          description: National Insurance number, in the format AA999999A.
+          type: string
+          example: TC663795B
+      /{taxYear}:
+        uriParameters:
+          taxYear:
+            description: 'The tax year to which the data applies. For example: 2022-23. The start year and end year must not span two tax years. The minimum tax year is 2022-23. No gaps are allowed, for example, 2022-24 is not valid. (The minimum tax year in Sandbox is 2021-22.)'
```

Comment:
I think a few people have missed this small detail - since this is a historic endpoint, in this instance it's not accurate to say that "The minimum tax year is 2022-23". The specs say that the *maximum* tax year is *2021-22*! Can we adjust this description and the example tax year on line 86 in light of this? 
<img width="700" alt="image" src="https://user-images.githubusercontent.com/106095575/180179847-cdb8b60f-7c48-47e1-9335-719a91955410.png">

Reply:
Good point